app/views/widgets/thermometer/_form.slim

```diff
@@ -0,0 +1,12 @@
+#thermometer_widget
+  div.form-group
+    label for='widgets[thermometer][goal]' Goal: 
+    input type='number' name='widgets[thermometer][goal]' value=10000
+  div.form-group
+    label for='widgets[thermometer][goal]' Starting count: 
+    input type='number' name='widgets[thermometer][count]' value=10000
```

Comment:
I think we should probably make this a lower default number than the goal, since that's an easy UX clue to suggest "this is the one that's supposed to be smaller". One of my most grating issues with ActionSweet is putting the goal and count in the wrong fields when I'm initializing a thermometer, and this would be an easy way to avoid that.